src/interface.rs

```diff
@@ -866,7 +857,7 @@ impl<'a> Interface<'a> {
 
                 KeyEvent {
                     modifiers: KeyModifiers::CONTROL,
-                    code: Char('c') | Char('g') | Char('z') | Char('r'), // TODO add ZZ as shortcut
+                    code: Char('c' | 'g' | 'z' | 'r'),
```

Comment:
Did you mean to remove this comment?

Reply:
You're right, the comment should be kept.